python/ray/parameter.py

```diff
@@ -73,6 +73,8 @@ class RayParams(object):
             Java worker.
         java_worker_options (str): The command options for Java worker.
         load_code_from_local: Whether load code from local file or from GCS.
+        enable_profiling: Whether profiling information for the timeline
```

Comment:
Why do you think it's a bad idea to have an option to turn off profiling? In my opinion, profiling should only be turned on when people want to look into profiling data. Otherwise, it's gonna generate tons of useless data, especially in a massive production env.

Reply:
If profiling is always enabled, then that will force us to optimize it and make sure it is fast, doesn't use infinite memory, and generally is stable in a production environment. If it can be disabled, then it will be less likely to get optimized. Users may have to choose between fast execution and helpful profiling information, and sometimes people will want both (e.g., if you want to profile an application where performance is really important, then you don't want the profiling to change the performance much).